Formula/ouch.rb

```diff
@@ -0,0 +1,32 @@
+class Ouch < Formula
+  desc "Painless compression and decompression for your terminal"
+  homepage "https://github.com/ouch-org/ouch"
+  url "https://github.com/ouch-org/ouch/archive/refs/tags/0.3.1.tar.gz"
+  sha256 "269abaf5ac2f80da3796dbf5e73419c1b64104d1295f3ff57965141f079e6f6d"
+  license "MIT"
+  head "https://github.com/ouch-org/ouch.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  uses_from_macos "bzip2"
+  uses_from_macos "xz"
+  uses_from_macos "zlib"
```

Comment:
Ah, yea, these are needed but aren't related to the build error.

It looks like `cargo` is trying to link statically to a just-build Zstd (😞) and failing miserably.